wordfreq/chinese.py

```diff
@@ -0,0 +1,24 @@
+from pkg_resources import resource_filename
+import jieba
+import msgpack
+import gzip
+
+jieba_tokenizer = None
+simplified_map = None
+DICT_FILENAME = resource_filename('wordfreq', 'data/jieba_zh.txt')
+SIMP_MAP_FILENAME = resource_filename('wordfreq', 'data/_chinese_mapping.msgpack.gz')
+
+
+def simplify_chinese(text):
+    global simplified_map
```

Comment:
Is this delayed load actually necessary?  It seems that in all cases where you even import this file you're going to end up here.